py3status/modules/arch_updates.py

```diff
@@ -14,8 +14,10 @@
 
 Requires:
     pacman-contrib: contributed scripts and tools for pacman systems
-    auracle: a flexible command line client for arch linux's user repository
-    yay: yet another yogurt. pacman wrapper and aur helper written in go
+    one of the following AUR helpers:
```

Comment:
We'll see if this works okay in readthedocs. Ty for cleaning up trailing whitespace.

Reply:
![image](https://user-images.githubusercontent.com/3451904/58995806-f6458800-87f5-11e9-846c-a348995e7ff0.png)
I'll change it back to the way it was.